### ISSUE TITLE
React DevTools should bump the Electron version to 11.0.1 for darwin-arm64 builds

### ISSUE THREAD

**Issue by mglukhovsky** [facebook/react#20454]
## Environment:
- macOS 11.0.1 Big Sur, on an Apple Silicon (M1) chip
- React 17.0.1
- React DevTools 4.10.1, running in standalone mode via Electron 11.1.0 (darwin-arm64 build)
- New project using create-react-app . with the useScript hook to add the <script> tag required by React DevTools.

## Steps to reproduce

1. Add React DevTools to a project with `npm install --save react-devtools` (or yarn).
2. react-devtools expects Electron 9.x.x; however builds for darwin-arm64 are only available for 11.1.0. Installation fails with:
```
npm ERR! code 1
npm ERR! path /Users/michael/git/components-blank/node_modules/electron
npm ERR! command failed
npm ERR! command sh -c node install.js
 HTTPError: Response code 404 (Not Found) for https://github.com/electron/electron/releases/download/v9.4.0/electron-v9.4.0-darwin-arm64.zip
 ...
```

Link to code example:  [mglukhovsky/react-devtools-components-blank](https://github.com/mglukhovsky/react-devtools-components-blank)

## The current behavior
Workarounds are [documented here](https://github.com/mglukhovsky/react-devtools-components-blank#electron-1110-required-for-m1); however a 11.1.0 build is required.

## The expected behavior
Bump the version in package.json so this works out of the box.

**Comment by bvaughn** [comment]
Want to submit a PR (and test the bump)?

**Comment by mdaj06** [comment]
I can look into this @bvaughn

**Comment by bvaughn** [comment]
Okay.

**Comment by kivikakk** [comment]
Just wanted to add that bumping the version works well.

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/1915/102738569-9f36e200-439e-11eb-89f1-874c45a08f03.png">

**Comment by jaiwanth-v** [comment]
Hi @mdaj06, are you working on this? If not, I'd like to take this issue.

**Comment by mdaj06** [comment]
Sure @jaiwanth-v im looking into another issue you can take it up!

**Comment by vladimirschneider** [comment]
I can update electron package version

**Comment by Raphael0147** [comment]
[devtools] Bump electron version from 9.1.0 to 11.1.0 for darwin-arm64 builds #20496

**Comment by vladimirschneider** [comment]
So, I think need close this issue @jaiwanth-v

**Comment by jaiwanth-v** [comment]
I think they'll close it once the PR is merged

**Comment by SaeedSheikhi** [comment]
How can we install this bumped version from github pull request? without waiting to be merged? @jaiwanth-v

**Comment by jaiwanth-v** [comment]
You'll have to clone my forked repo and build from source. You can find the instructions to build [here](https://www.npmjs.com/package/react-devtools#local-development)

**Comment by badal897** [comment]
Can someone please tell how to use resolutions ? I am macbook pro m1 and when I hit the command ->
`npm i -g react-devtools@"<4.11.0"`

It gives the following error
`HTTPError: Response code 404 (Not Found) for https://github.com/electron/electron/releases/download/v9.4.4/electron-v9.4.4-darwin-arm64.zip`

What I tried is I downloaded the react devtools 4.10.1 in my local system and tried install it using
`npm i -g 'path to downloaded repo'`

Then I installed react-native-debugger and then again got the same issue.

**Unsupported DevTools backend version
You are running react-devtools version 4.13.4-51ebccc374.
This requires bridge protocol version 1. However the current backend version uses bridge protocol version 0.**

To fix this, downgrade the DevTools NPM package:
`npm i -g react-devtools@"<4.11.0"`

**Comment by alliuca** [comment]
My dirty workaround, things seem to work fine on my M1 machine:

- Install Electron 11 globally `npm i -g electron@11.1.0`
- Install react-devtools without the binary download `ELECTRON_SKIP_BINARY_DOWNLOAD=1 npm i -g react-devtools@"<4.11.0"`
- Remove the existing `dist` folder and `path.txt` in the react-devtools electro module
  - `rm -rf $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron/dist`
  - `rm -rf $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron/path.txt`
- Symlink the global electron installation to the react-devtools electron module installation
  - `ln -s $(dirname $(dirname $(which node)))/lib/node_modules/electron/dist $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron`
  - `ln -s $(dirname $(dirname $(which node)))/lib/node_modules/electron/path.txt $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron`
  
 I also had to update the `react-devtools-core` package inside my global `react-devtools` installation to 4.11.0 (could use the same symlinking logic there too I guess)

**Comment by estebanmino** [comment]
thanks @alliuca! for me, just the first two steps worked on my M1 machine
- Install Electron 11 globally npm i -g electron@11.1.0
- Install react-devtools without the binary download ELECTRON_SKIP_BINARY_DOWNLOAD=1 npm i -g react-devtools@"<4.11.0"

**Comment by nicholasdunbar** [comment]
I had to follow the steps in https://gist.github.com/bvaughn/4bc90775530873fdf8e7ade4a039e579 mixed with @alliuca 's steps. 

I'm using a mac M1 which `npm i -g react-devtools@"<4.13.0"` gives the error `HTTPError: Response code 404 (Not Found) for https://github.com/electron/electron/releases/download/v9.4.4/electron-v9.4.4-darwin-arm64.zip` 

So I used the following modified steps:  

`npm uninstall -g react-devtools`
`npm i -g electron@11.1.0`
`yarn add -D react-devtools-core@4.13.0`
`ELECTRON_SKIP_BINARY_DOWNLOAD=1 npm i -g react-devtools@"<4.13.0"`
`rm -rf $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron/dist && rm -rf $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron/path.txt && ln -s $(dirname $(dirname $(which node)))/lib/node_modules/electron/dist $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron && ln -s $(dirname $(dirname $(which node)))/lib/node_modules/electron/path.txt $(dirname $(dirname $(which node)))/lib/node_modules/react-devtools/node_modules/electron`
